test/acceptance/graphql_resolvers/local_get_with_unlimited_vector_search.go

```diff
@@ -0,0 +1,338 @@
+					}
+				) {
+					_additional {
+						vector
+					}
+					contents
+				}
+			}
+		}
+		`
+		result := AssertGraphQL(t, helper.RootAuth, query)
+		notes := result.Get("Get", "RansomNote").AsSlice()
+		require.NotEmpty(t, notes)
+		require.Equal(t, 10, len(notes))
+	})
+
+	t.Run("nearVector: results limited by certainty", func(t *testing.T) {
```

Comment:
@etiennedi unlimited search , limited by certainty (no limit passed, but certainty set very high to limit results to less than default limit)